tests/e2e/csi_snapshot_basic.go

```diff
@@ -4706,6 +4826,9 @@ var _ = ginkgo.Describe("[block-vanilla-snapshot] Volume Snapshot Basic Test", f
 				err := snapc.SnapshotV1().VolumeSnapshots(namespace).Delete(ctx,
 					snapName, metav1.DeleteOptions{})
 				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
```

Comment:
whereever you are deleting the snapshot you can use wait for snapshot deletion in CNS instead of this wait

Reply:
I have added a check to verify snapshot deletion in CNS  and due to intermittent test failures the wait with full sync is needed